crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -556,21 +556,6 @@ pub(crate) async fn validate_op(
     }
 }
 
-/// For now errors result in an outcome but in the future
-/// we might find it useful to include the reason something
-/// was rejected etc.
-/// This is why the errors contain data but is currently unread.
-fn handle_failed(error: &ValidationOutcome) -> Outcome {
-    use Outcome::*;
-    match error {
-        ValidationOutcome::CounterfeitAction(_, _) => {
-            unreachable!("Counterfeit ops are dropped before sys validation")
-        }
-        ValidationOutcome::DepMissingFromDht(_) => MissingDhtDep,
-        reason => Rejected(reason.to_string()),
-    }
-}
-
```

Comment:
This function isn't used anywhere and the Rust compiler warns about that too. Can it be deleted?

Reply:
Sure, I'm surprised that it's not used, but if it ain't, it ain't.